infogami/infobase/_dbstore/sequence.py

```diff
@@ -14,16 +15,20 @@ def fire_event(self, event_name, name, value):
 
     def get_value(self, name):
         try:
-            return self.db.query("SELECT * FROM seq WHERE name=$name", vars=locals())[0].value
+            return self.db.query("SELECT * FROM seq WHERE name=$name", vars=locals())[
+                0
+            ].value
```

Comment:
This one is just funny 😂

Reply:
# fmt: off 
_code that I do not want reformatted_
# fmt: on